packages/editor/src/components/post-last-revision/index.js

```diff
@@ -11,22 +11,27 @@ import { addQueryArgs } from '@wordpress/url';
  * Internal dependencies
  */
 import PostLastRevisionCheck from './check';
+import PostPanelRow from '../post-panel-row';
 import { store as editorStore } from '../../store';
 
-/**
- * Renders the component for displaying the last revision of a post.
- *
- * @return {Component} The component to be rendered.
- */
-function PostLastRevision() {
-	const { lastRevisionId, revisionsCount } = useSelect( ( select ) => {
+function usePostLastRevisionInfo() {
+	return useSelect( ( select ) => {
 		const { getCurrentPostLastRevisionId, getCurrentPostRevisionsCount } =
 			select( editorStore );
 		return {
 			lastRevisionId: getCurrentPostLastRevisionId(),
 			revisionsCount: getCurrentPostRevisionsCount(),
 		};
 	}, [] );
+}
+
+/**
+ * Renders the component for displaying the last revision of a post.
+ *
+ * @return {Component} The component to be rendered.
+ */
+function PostLastRevision() {
```

Comment:
I don't think it's the same case because here we're not changing the entire panel (`PostLastRevisionPanel`) that is exported. So consumers of the above panel, should have the 'old' `PostLastRevision` component.

Reply:
If the policy is to keep the old components intact, then we might need to revisit some of the refactoring, like #61357. Because `PostDiscussionPanel` has a new structure.

Reply:
I don't think there is a clear policy about that, but I think the changes should not be disruptive in the sense that they would make sense in other designs too. An example would be if we change a control and the exporting component is just that. In this case IMO it's fine to not keep the old version. In this PR for me it made more sense to keep the old one as it matches better when used inside `PostLastRevisionPanel` with the panel body and we actually don't use `PostLastRevisionPanel` anymore in our codebase.

Can you share the components you're concerned with the discussions panel? Also cc @jorgefilipecosta for visibility.

Reply:
My concern is that the discussion panel consumers expect it to render `PanelBody > PanelRow`. Now, they'll get a different design.

I personally don't mind either path for changes as long as they're consistent.

Reply:
I know they get a different design and might be a bit different, but in my mind they still get the whole functionality (the whole panel). If I had to update the panel here for revisions personally I'd follow the same approach with the discussions panel. This is subjective though..